### ISSUE TITLE
Angular 1.5 is replaced by angular 1.3 due to this package

### ISSUE THREAD

**Issue by sunilpes** [planet-training/meteor-angular-ui-calendar#2]
In my project, i have installed angular 1.5.7 through NPM. But when i add this meteor package, the angular package in my project was replaced by 1.3.15. Please fix this

**Comment by mxab** [comment]
If you're using NPM already why not also use fullcalendar via npm as well?

**Comment by sunilpes** [comment]
yes, that solves the problem. But this package has created problem due to that. Any solution we can figure  it out?  we cannot restrict others to use meteor package along with NPM.
